.eslintrc.json

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
```

Comment:
Should this be excluded given we are building for React Native instead of the browser?

Reply:
There doesn't seem to be consensus online regarding this, but I think it's best that we exclude both browser and node, and replace es6 with es2020 (because there's no reason to constrain ourselves to older syntax - react native comes with babel built in). We should then also update ecmaVersion under parserOptions

Reply:
Sounds good